qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageAppointmentsPage.java

```diff
@@ -15,14 +15,14 @@
 public class ManageAppointmentsPage extends Page {
 
 	public static final By DELETE_REQUEST = By.cssSelector(".delete-item.icon-remove");
-	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons > button");
-	private static final By SEARCH = By.cssSelector("#searchButtons > button.confirm");
-	private static final By APPOINTMENT = By.xpath("//table[@id='appointmentTable']/div[2]/div/div/div/div[2]/div[2]/div");
-	private static final By NEXT = By.cssSelector("#selectAppointment > button.confirm");
-	private static final By SAVE = By.xpath("//div[@id='confirmAppointment']/div[2]/button[2]");
+	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons button");
+	private static final By SEARCH = By.cssSelector("#searchButtons button.confirm");
+	private static final By APPOINTMENT = By.cssSelector("#appointmentTable div:nth-child(2) div:nth-child(1) div:nth-child(1) div:nth-child(1) div:nth-child(2) div:nth-child(2) div:nth-child(1)");
+	private static final By NEXT = By.cssSelector("#selectAppointment button.confirm");
+	private static final By SAVE = By.cssSelector("#confirmAppointment div:nth-child(3) button.confirm.right");
```

Comment:
even here

Reply:
@kdaud  Let me look out for other selectors